job-server/src/main/scala/spark/jobserver/BinaryManager.scala

```diff
@@ -56,10 +61,17 @@ class BinaryManager(jobDao: ActorRef) extends InstrumentedActor {
   }
 
   private def deleteBinary(appName: String): Future[Try[Unit]] = {
-    (jobDao ? JobDAOActor.DeleteBinary(appName)).
+    (jobDao ? JobDAOActor.DeleteBinary(appName))(BinaryManager.DELETE_TIMEOUT).
       mapTo[DeleteBinaryResult].map(_.outcome)
   }
 
+  private def getActiveJobsUsingBinary(binName: String): Future[Seq[JobInfo]] = {
+    (jobDao ? JobDAOActor.GetJobsByBinaryName(
+          binName, Some(JobStatus.getNonFinalStates())))(BinaryManager.DELETE_TIMEOUT)
```

Comment:
I think it should be renamed to `JobStatus.nonFinalStates`.... get is redundant

Reply:
ok. Would try to address this in a separate change.